README.md

```diff
@@ -27,7 +70,15 @@ PSPDFKit - The Leading Mobile PDF Framework for iOS and Android.
 </ti:app>
 ```
 
-Note: PSPDFKit 8 for iOS needs at least Xcode 10.2 or higher and supports iOS 11+.
+### Using the correct version of Titanium
+
+If you have multiple versions of the Titanium SDK installed on your system, you'll need to also modify the `titanium.xcconfig` configuration file to set the correct version number:
+
+```
+TITANIUM_SDK_VERSION = 8.2.1.GA
+```
+
+If you do not do this, have multiple versions of the Titanium SDK installed, and none of them is the one that's set on the `xcconfig` file, the build will fail with cryptic error messages.
```

Comment:
Is this necessary for the end user? We already do this in `titanium.xcconfig`.

Reply:
Faced an issue where I couldn't get the thing to compile because I had a more recent version installed, and was a bit of digging. It's not _necessary_, but a good pointer to give, I'd say.

Reply:
Makes sense1 Let's add this under the Troubleshooting section then.